build.sbt

```diff
@@ -18,13 +16,10 @@ crossVersion := CrossVersion.binary
 val amzVersion = "1.12.286"
 val testcontainersScalaVersion = "0.40.10"
 
-// TODO: Remove when Pekko has a proper release
-resolvers += Resolver.ApacheMavenSnapshotsRepo
-updateOptions := updateOptions.value.withLatestSnapshots(false)
-
 ThisBuild / apacheSonatypeProjectProfile := "pekko"
 ThisBuild / versionScheme := Some(VersionScheme.SemVerSpec)
-sourceDistName := "incubating-pekko-persistence-dynamodb"
+sourceDistName := "apache-pekko-persistence-dynamodb"
```

Comment:
Does the `apache-` is required?

Reply:
was requested in https://lists.apache.org/thread/nyn2kmfhln27vxtybyjq2vb69tk1hjgw